crates/circuit/src/dag_circuit.rs

```diff
@@ -43,6 +43,11 @@ use rustworkx_core::petgraph::prelude::StableDiGraph;
 use rustworkx_core::petgraph::stable_graph::{DefaultIx, IndexType, Neighbors, NodeIndex};
 use rustworkx_core::petgraph::visit::{IntoNodeReferences, NodeCount, NodeRef};
 use rustworkx_core::petgraph::Incoming;
+use rustworkx_core::traversal::{
```

Comment:
Seeing this here, I think we should probably alias `rustworkx_core` as `rx` and then use fully qualified paths to make the callsites cleaner, i.e.

```rust
use rustworkx_core as rx;
```

And then at the call site, do `rx::traversal::ancestors(...)`.

No action needed, I can adjust after merge.